config/ldes-delta-pusher/dispatch.ts

```diff
@@ -0,0 +1,59 @@
+import { moveTriples } from "../support";
+import { Changeset } from "../types";
+import { query, sparqlEscapeUri } from 'mu';
+
+export default async function dispatch(changesets: Changeset[]) {
+	for (const changeset of changesets) {
+		const subjects = new Set(changeset.inserts.map((insert) => insert.subject.value));
+		for (const subject of subjects) {
+			const { results: { bindings } } = await query(`
+        PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+        PREFIX code: <http://telegraphis.net/ontology/measurement/code#>
+        PREFIX org: <http://www.w3.org/ns/org#>
+        PREFIX adms: <http://www.w3.org/ns/adms#>
+  
+        CONSTRUCT {
+          ?organization a org:Organization;
```

Comment:
I tested the PR by updating a phone number in a municipality. I didnt see any file generated, so I did a (very small) investigation, I found that the ldes-backend wasn't getting any activity, and when running manually the queries in Virtuoso I saw that enforcing the type `org:Organization` seemed to return nothing because Virtuoso isn't aware of inheritance. But if the query goes through mu-cl-resources it should I think, so I'm not sure why the backend doesn't receive anything. Can you describe the happy-path you use for testing ? Thank you :)

Reply:
Yes I now see it doesn't return anything anymore while it did on the previous version. I could try specifying the necessary subtypes in the query.

Reply:
So using the migration and for newly created organizations, organizations should have the org:Organization type. When creating/editing an organization and its contact info, they should be added to the `organizations` ldes-feed. The first page should be accessible through `http://localhost/ldes/organizations/1`